### ISSUE TITLE
No /usr/local/jails/modules/forms.d/natcfg/initforms.sh

### ISSUE THREAD

**Issue by trombik** [cbsd/cbsd#134]
when choosing `natcfg` in `initenv-tui`, the dialog does not seem to work.

## how to reproduce

* `pkg install cbsd`
* set `nodename`
* run `/usr/local/cbsd/sudoexec/initenv inter=0`

```
cbsd@server1> uname -msr
FreeBSD 10.3-RELEASE amd64
root@server1:/usr/local/etc # pkg info | grep cbsd
cbsd-11.0.6_1                  Yet another FreeBSD Jail and Bhyve Management Utility
root@server1:/usr/local/etc # env workdir=/usr/local/jails cbsd
cbsd@server1> initenv-tui
... choose [natcfg] in the dialog ...
No /usr/local/jails/modules/forms.d/natcfg/initforms.sh
root@server1:/usr/local/etc # find /usr/local/cbsd | grep initforms.sh 
root@server1:/usr/local/etc # find /usr/local/jails | grep initforms.sh
root@server1:/usr/local/etc # 
```

**Comment by olevole** [comment]
Ugh. TUI dialog for NAT is new functional and still not ready, forget to disable in initenv ;-)
It will be fixed in next release,  11.0.9 or 11.1.0

**Comment by kaN5300** [comment]
```
cbsd-11.1.6
Name           : cbsd
Version        : 11.1.6
Installed on   : Mon Sep 25 15:34:31 2017 UTC
Origin         : sysutils/cbsd
Architecture   : FreeBSD:11:amd64
Prefix         : /usr/local
Categories     : sysutils
Licenses       : BSD2CLAUSE
Maintainer     : olevole@olevole.ru
WWW            : https://www.bsdstore.ru
Comment        : Yet another FreeBSD Jail and Bhyve Management Utility
Shared Libs required:
	libedit.so.0
	libssh2.so.1
	libsqlite3.so.0
Annotations    :
	repo_type      : binary
	repository     : FreeBSD
Flat size      : 16.0MiB
Description    :
Yet another FreeBSD jail and bhyve management tool.

WWW: https://www.bsdstore.ru

```

```
cbsd@cbsd> natcfg-tui
No /usr/jails/modules/forms.d/natcfg/initforms.sh
```

What is the proper way to organize some dst nat rules for jails with ipfw? There is ${etcdir}/ipfw.conf witch in my case consists of 3 lines:

...
/sbin/ipfw -q add 2000 nat 123 all from 10.0.0.0/8,172.16.0.0/12,192.168.0.0/16 to not 10.0.0.0/8,172.16.0.0/12,192.168.0.0/16 any via em0
/sbin/ipfw -q nat 123 config ip 10.0.2.15
/sbin/ipfw -q add 2000 nat 123 ip from any to 10.0.2.15 via em0
...

**Comment by olevole** [comment]
@kaN5300 The contents of this file are fully automatic. If you want to have your own rules, you need to turn off the NAT function in CBSD ( `cbsd initenv-tui` or `cbsd natcfg` + `cbsd natoff`). 

And move  these rules to your regular firewall configuration file.
To be able to mix the CBSD firewall rules together with the user's - this is a good feature. However at the moment I do not have a clear understanding/view of how to do it the best way since it is a rather dangerous component.
Perhaps this should be implemented as separate blocks that are concat together in a certain order in the final file.

**Comment by olevole** [comment]
fixed in CBSD 11.1.12

**Comment by olevole** [comment]
11.1.12 commited into ports tree
